docs/releases/4.0.txt

```diff
@@ -263,6 +263,8 @@ Models
 * :class:`~django.db.models.DurationField` now supports multiplying and
   dividing by scalar values on SQLite.
 
+* :meth:`.QuerySet.bulk_update` now returns the count of the objects updated.
```

Comment:
`count` -> `number`
`objects updated` -> `updated objects`

```suggestion
* :meth:`.QuerySet.bulk_update` now returns the number of objects updated.
```

Reply:
i have made the change as per your suggestion but the phrase `returns the number of objects updated` sounds more to me as something that returns the objects, maybe count seemed more appropriate here. although, i may be pedantic and wrong here.

Reply:
The phrasing Mariusz suggested is correct.

Reply:
okay, thanks for reviewing this up.